### DISCUSSION TITLE
Document upload format- PDF to be added

### DISCUSSION THREAD

**Discussion by Kaustubh804975** [GitbookIO/community#371] [Feature Requests]
Hi Team,

Please look out for the possibility of uploading PDF document feature as we upload word document.

Thanks

**Comment by taszap** [comment]
Hi @Kaustubh804975 thank you for your feedback. Sadly we have no plans of shipping this in the near future. 
As a workaround, we would recommend converting PDFs into other formats accepted by GitBook. 

If anything changes we will definitely update you here
